hack/run-ci-e2e-test.sh

```diff
@@ -25,13 +25,13 @@ set -o monitor
 RUNNING_IN_CI=${JOB_NAME:-""}
 BUILD_ID=${BUILD_ID:-"${USER}-local"}
 PROVIDER=${PROVIDER:-"aws"}
-TERRAFORM_VERSION=${TERRAFORM_VERSION:-"0.12.5"}
+TERRAFORM_VERSION=${TERRAFORM_VERSION:-"0.12.13"}
```

Comment:
This doesn't have any effect as we never download Terraform because it's already in the image.